transformer_engine/pytorch/csrc/comm_gemm_overlap.h

```diff
@@ -95,6 +95,9 @@ struct UbufCommOverlap : torch::CustomClassHolder, UbufBase {
     }
     _ubuf = torch::from_blob(_ubuf_ptr, {sample.size(0), sample.size(1)}, sample.options());
 
+    // FIXME: We no longer rely on NVTE_UB_ATOMIC_GEMM_RS, so we cannot determine
```

Comment:
PR is ready for review as it passed testing.
Could you advise on what to do regarding this print code block?
CC @ksivaman

Reply:
I think it's fine to remove this since it's just debugging messages.